tests/api/test_devicespec.py

```diff
@@ -83,17 +92,18 @@
 }
 
 
-class TestDeviceSpec:
-    """Tests for the ``DeviceSpec`` class."""
+class TestDevice:
+    """Tests for the ``Device`` class."""
 
     def test_initialization(self):
         """Test that the device spec class initializes correctly."""
-        spec = DeviceSpec(spec=device_dict)
+        device = Device(spec=device_dict, certificate=device_certificate)
```

Comment:
Should we rename this too while we're at it?
```suggestion
        device = Device(spec=device_spec, certificate=device_certificate)
```